javascript-practice/src/server/db.json

```diff
@@ -1,196 +1,178 @@
 			},
-			"image": ""
+			"image": "/src/assets/images/graystone-vase.png"
 		},
 		{
 			"id": 4,
 			"title": "Small vase",
 			"price": 250,
 			"description": "A ceramic is an inorganic non-metallic solid made up of either metal or non-metal compounds that have been shaped and then hardened by heating to high temperatures.",
-			"category": {
-				"id": 1,
-				"name": "Ceramics"
-			},
+			"categoryId": 2,
 			"dimension": {
 				"height": "50cm",
 				"width": "10cm",
 				"depth": "2cm"
 			},
-			"image": ""
+			"image": "/src/assets/images/small-vase.png"
 		},
 		{
 			"id": 5,
 			"title": "The Lucy Lamp",
 			"price": 399,
 			"description": "A lamp is a device that makes light and heat. Lamps usually work with electricity, using a lightbulb.",
-			"category": {
-				"id": 2,
-				"name": "Ligth fittings"
-			},
+			"categoryId": 7,
 			"dimension": {
 				"height": "50cm",
 				"width": "10cm",
 				"depth": "2cm"
 			},
-			"image": ""
+			"image": "/src/assets/images/lamp.png"
 		},
 		{
 			"id": 6,
 			"title": "Multiple Lamps",
 			"price": 399,
 			"description": "A lamp is a device that makes light and heat. Lamps usually work with electricity, using a lightbulb.",
-			"category": {
-				"id": 2,
-				"name": "Ligth fittings"
-			},
+			"categoryId": 7,
 			"dimension": {
 				"height": "50cm",
 				"width": "10cm",
 				"depth": "2cm"
 			},
-			"image": ""
+			"image": "/src/assets/images/multiple-lamp.png"
 		},
 		{
 			"id": 7,
 			"title": "The Dandy chair",
 			"price": 250,
 			"description": "A timeless design, with premium materials features as one of our most popular and iconic pieces. The dandy chair is perfect for any stylish living space with beech legs and lambskin leather upholstery.",
-			"category": {
-				"id": 3,
-				"name": "Chairs"
-			},
+			"categoryId": 4,
 			"dimension": {
 				"height": "110cm",
 				"width": "75cm",
 				"depth": "50cm"
 			},
-			"image": ""
+			"image": "/src/assets/images/chair.png"
 		},
 		{
 			"id": 8,
-			"title": "The Dandy chair",
-			"price": 250,
-			"description": "A timeless design, with premium materials features as one of our most popular and iconic pieces. The dandy chair is perfect for any stylish living space with beech legs and lambskin leather upholstery.",
-			"category": {
-				"id": 3,
-				"name": "Chairs"
-			},
-			"dimension": {
-				"height": "110cm",
-				"width": "75cm",
-				"depth": "50cm"
-			},
-			"image": ""
-		},
-		{
-			"id": 9,
 			"title": "The Dark chair",
 			"price": 250,
 			"description": "A timeless design, with premium materials features as one of our most popular and iconic pieces. The dandy chair is perfect for any stylish living space with beech legs and lambskin leather upholstery.",
-			"category": {
-				"id": 3,
-				"name": "Chairs"
-			},
+			"categoryId": 4,
 			"dimension": {
 				"height": "110cm",
 				"width": "75cm",
 				"depth": "50cm"
 			},
-			"image": ""
+			"image": "/src/assets/images/dark-chair.png"
```

Comment:
About image, we can use as an URL will be better than using the image from asssets

Reply:
You can input the URL from internet @mthanh2209

Reply:
Thank you.